packages/passport-interface/src/RequestTypes.ts

```diff
@@ -135,26 +105,228 @@ export type TicketError =
   | { name: "InvalidTicket" }
   | { name: "TicketRevoked"; revokedTimestamp: number }
   | { name: "NetworkError" }
```

Comment:
I like the separation of error types here.  The generic ones here (NetworkError and ServerError) should probably be separated out for use in other response types?